cli/flox/src/config/mod.rs

```diff
@@ -93,6 +93,9 @@ pub struct FloxConfig {
 
     /// Hide environments named 'default' from the shell prompt
     pub hide_default_prompt: Option<bool>,
+
+    /// Check for upgrades on `flox activate` (default: true)
```

Comment:
blocking: this implies the check isn't performed
```suggestion
    /// Print notification if upgrades are available on `flox activate` (default: true)
```